cli/.coveragerc

```diff
@@ -0,0 +1,5 @@
+[run]
+omit =
+    **/__init__.py
+    ripsaw/cli.py
```

Comment:
Why are we ignoring the CLI and ``commands/`` sub-dir? Are those out of scope for the unit tests?

Reply:
for unit tests, yeah. The `cli.py` file is just the code entrypoint and only really executes if you run the cli itself. `commands` I will remove from there because even though it's out of scope for unit tests we shouldn't ignore it from coverage

Reply:
Sounds good! Thanks for the clarification.